### PR TITLE
[TT-1869] Add link to Gateway API Page

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ menu:
 weight: 95
 ---
 
-The Tyk Gateway REST API is the primary means for integrating your application with the Tyk API Gateway system. This
+The [Tyk Gateway API](/docs/tyk-gateway-api/) is the primary means for integrating your application with the Tyk API Gateway system. This
```
Comment (letzya) [inline tyk-docs/content/getting-started/key-concepts/gateway-api.md]: i'd add "REST" - `The Tyk-Gateway RESP APIs are the primary means...`

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ The Tyk Gateway API is capable of:
 * Hot reloads / reloading a cluster configuration
 * OAuth client creation (*only* when not using the Dashboard)
 
-In order to use the REST API, you'll need to set the `secret` parameter in your `tyk.conf` file.
+In order to use the Gateway API, you'll need to set the `secret` parameter in your `tyk.conf` file.
```
Comment (letzya) [inline tyk-docs/content/getting-started/key-concepts/gateway-api.md]: "In order to use the Tyk getaway REST APIs,"

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md]: Updated

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md]: Updated

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ The Tyk Gateway API is capable of:
 * Hot reloads / reloading a cluster configuration
 * OAuth client creation (*only* when not using the Dashboard)
 
-In order to use the REST API, you'll need to set the `secret` parameter in your `tyk.conf` file.
+In order to use the Tyk Gateway REST API, you'll need to set the `secret` parameter in your `tyk.conf` file.
```
Comment (letzya) [inline tyk-docs/content/getting-started/key-concepts/gateway-api.md:29]: APIs

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ menu:
 weight: 95
 ---
 
-The Tyk Gateway REST API is the primary means for integrating your application with the Tyk API Gateway system. This
+The [Tyk Gateway REST API](/docs/tyk-gateway-api/) is the primary means for integrating your application with the Tyk API management platform. This
```
Comment (letzya) [inline tyk-docs/content/getting-started/key-concepts/gateway-api.md]: it's not a management platform

Comment (marksou) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md]: It's how we've been referring to it in the restructure. See last paragraph here - https://tyk.io/docs/apim/open-source/#tyk-gateway

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md (hunk 1 above)
Comment (letzya) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md]: Why cant I see the update @marksou

**Review by letzya — CHANGES_REQUESTED** [review]
Sorry, couldn't see the changes

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md (hunk 3 above)
Comment (marksou) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md:29]: There is only one Gateway API?

**Comment by marksou** [comment]
@letzya click on the second commit.

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md (hunk 3 above)
Comment (letzya) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md:29]: /debug and /hello are 2 more APIs. 
But actually, you need the secret only for /tyk api.

File: tyk-docs/content/getting-started/key-concepts/gateway-api.md (hunk 4 above)
Comment (letzya) [reply tyk-docs/content/getting-started/key-concepts/gateway-api.md]: @marksou not sure I understand. can you add the quote here or just leave this as it's currently written in master and just add a link to the swagger?

**Review by letzya — APPROVED** [review]

**Pull request merged**